.github/workflows/validate-linux-binaries.yml

```diff
@@ -36,10 +36,8 @@ jobs:
       fail-fast: false
     runs-on: ${{ matrix.validation_runner }}
     steps:
-      - name: Checkout PyTorch builder
-        uses: actions/checkout@v2
       - name: Validate binary conda
-        uses: ./.github/actions/validate-linux-binary
+        uses: pytorch/builder/.github/actions/validate-binary@main
```

Comment:
Are you worried that using a not local version of this action will make it harder to roll out new changes?

Reply:
As per discussion will move to local action rather then global one!